### ISSUE TITLE
Add accessibility information to the Flutter for SwiftUI developers guide

### ISSUE THREAD

**Issue by leighajarett** [flutter/website#8652]
### What information is missing?

We should add some examples of how [SwiftUI accessibility modifiers](https://developer.apple.com/documentation/swiftui/view-accessibility) map to Flutter's accessibility features. 

Page link: https://docs.flutter.dev/get-started/flutter-for/swiftui-devs
